ouroboros-consensus/src/Ouroboros/Consensus/Util/MonadSTM/StrictMVar.hs

```diff
@@ -5,6 +5,7 @@
 {-# LANGUAGE TupleSections     #-}
 {-# LANGUAGE TypeFamilies      #-}
 
+-- TODO: this module ought to use 'MonadMVar', and be moved to `strict-stm`.
```

Comment:
It'd help whoever needs to do this to provide a rationale for this.

Reply:
I updated the todo.